src/pages/content/word-list.mdx

```diff
@@ -1,12 +1,707 @@
 ---
 title: Word list
-description: >
-  Lorem ipsum dolor sit amet, consectetur adipiscing
-  elit, sed do eiusmod tempor incididunt ut labore et
-  dolore magna aliqua.
 ---
 
-Coming soon
+## Words we use
+
+### @mention
+
+Pronounced “at mention.” Always lowercase, even at the beginning of a sentence.
+
+### accessible, accessibility, a11y
+
+Describes a set of standards that make our product usable to people experiencing
+disabilities. Avoid using accessible as a synonym for “simple.” For that
+purpose, use terms like easy to learn, easy to use, or intuitive.
+
+### add
+
+Use add when moving a “thing” in the product from one place to another.
```

Comment:
Needs italics:

Use _add_ when moving a “thing” in the product from one place to another.